moonraker/components/octoprint_compat.py

```diff
@@ -43,6 +43,15 @@ def __init__(self, config: ConfigHelper) -> None:
             'software_version')
         self.enable_ufp: bool = config.getboolean('enable_ufp', True)
 
+        # Get webcam settings from config
+        self.webcam: Dict[str, Any] = {
+            'flipH': config.getboolean('flipH', False),
+            'flipV': config.getboolean('flipV', False),
+            'rotate90': config.getboolean('rotate90', False),
+            'streamUrl': config.get('streamUrl', '/webcam/?action=stream'),
+            'webcamEnabled': config.getboolean('webcamEnabled', True),
```

Comment:
Moonraker's configuration options are standardized to use snake case.  They should be something like:
 ```
flip_h: 
flip_v
rotate_90:
stream_url:
webcam_enabled:
```
Its fine for the dict keys to use camel case since that is what the response requires.